analytics.py

```diff
@@ -0,0 +1,117 @@
+"""
+By Eric Friesenhahn

```

Comment:
You can use `__authors__` as well.  Then someone could make a call: `analytics.__authors__` and get your name.  Another magic method (in a module).